### DISCUSSION TITLE
ET-2400 support?

### DISCUSSION THREAD

**Discussion by jakusotsu** [Ircama/epson_print_conf#3] [Ideas]
Hi, ET-2400 is very affordable right now so was hoping if this utility could support it. 
CTRL+F could not find ET-2400 in the conf file.

**Comment by dgerber** [comment]
> Hi, ET-2400 is very affordable right now so was hoping if this utility could support it.
> CTRL+F could not find ET-2400 in the conf file.

See <https://codeberg.org/atufi/reinkpy/src/branch/main/reinkpy/epson.toml>. It
may be the same as ET-2720.

**Comment by Ircama** [comment]
ET-2400 added.
